client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -532,6 +532,8 @@ export const FeatureRow = React.memo(
     ) : (
       <code>{name}</code>
     );
+    const path = name.split(".");
+    const depth = path.length;
```

Comment:
I was hoping that this allows us to differentiate between first-, second- and third-level features, but it turns out it doesn't, because only third-level features (like `global.prototype_accessor`) have a dot, so this returns a depth of 1 for both `RegExp` (first-level) and `global` (second-level).

Maybe it would make sense to extract a method `getFeatureDepth()` as follows:

```ts
function getFeatureDepth({ name, isRoot }: {
      name: string;
      isRoot: boolean;
}) {
  if (name.includes('.')) {
    const path = name.split('.');

    return path.length;
  }

  return isRoot ? 0 : 1;
}
```

Reply:
Then it would look like this, which I find pretty neat:

<img width="779" alt="image" src="https://user-images.githubusercontent.com/495429/161055491-f0d73c15-0870-43af-9afc-87c6cf0426ff.png">

Reply:
> For the record, we already have internal consensus for intending the third level as follows, but I will discuss the version above as well

Once you've discussed this, let me know if I should add this function.

Reply:
So what's the thought here then. My thought would be:

Replace the above line with:

`const depthClass = name.includes('.') ? 'bc-feature-depth-3' : '';`

Then, replace ``<th className={`bc-feature bc-feature-depth-${depth}`} scope="row">`` with ``<th className={`bc-feature ${depthClass}`} scope="row">`` and replace:

```
  .bc-feature-depth-2 {
    border-left-width: 8px;
  }

  .bc-feature-depth-3 {
    border-left-width: 16px;
  }
```

with

```
  .bc-feature-depth-3 {
    border-left-width: 8px;
  }
```

Is there a different preference?

Reply:
Hi @danielhjacobs, I'm not sure if you saw my comment in https://github.com/mdn/yari/pull/5884#issuecomment-1088855412?

The tl;dr is @queengooborg replaced `isRoot` by `depth`, which is merged and now at your disposal, once you merge `main` into your branch.

Your PR would then just need to add `bc-feature-depth-${depth}` to all `th.bc-feature` cells and add the CSS rule for `bc-feature-depth-2` (0-indexed), and that should be all. :)